scripts/deploy.sh

```diff
@@ -87,104 +47,69 @@ then
 fi
 
-# Printing the ownership
-print_title "Checking contracts ownership"
-echo "DripsHub admin:   $(cast call "$DRIPS_HUB" 'admin()(address)')"
-
-# Building and printing the deployment JSON
-print_title "Deployment JSON: $DEPLOYMENT_JSON"
+# Build and print the deployment JSON
+print_title Building the deployment JSON: "$DEPLOYMENT_JSON"
 tee "$DEPLOYMENT_JSON" <<EOF
 {
-    "Network":                      "$NETWORK",
-    "Deployer address":             "$DEPLOYER",
-    "Caller":                       "$CALLER",
-    "DripsHub":                     "$DRIPS_HUB",
-    "DripsHub logic":               "$DRIPS_HUB_LOGIC",
-    "DripsHub cycle seconds":       "$CYCLE_SECS",
-    "AddressDriver":                "$ADDRESS_DRIVER",
-    "AddressDriver logic":          "$ADDRESS_DRIVER_LOGIC",
-    "AddressDriver ID":             "$ADDRESS_DRIVER_ID",
-    "NFTDriver":                    "$NFT_DRIVER",
-    "NFTDriver logic":              "$NFT_DRIVER_LOGIC",
-    "NFTDriver ID":                 "$NFT_DRIVER_ID",
-    "ImmutableSplitsDriver":        "$SPLITS_DRIVER",
-    "ImmutableSplitsDriver logic":  "$SPLITS_DRIVER_LOGIC",
-    "ImmutableSplitsDriver ID":     "$SPLITS_DRIVER_ID",
+    "Chain":                        "$CHAIN",
+    "Wallet":                       "$WALLET",
+    "Wallet nonce":                 "$WALLET_NONCE",
+    "Deployer":                     "$DEPLOYER",
+    "DripsHub":                     "$(deployment_detail dripsHub address)",
+    "DripsHub cycle seconds":       "$(deployment_detail dripsHubCycleSecs uint32)",
+    "DripsHub logic":               "$(deployment_detail dripsHubLogic address)",
+    "DripsHub admin":               "$(deployment_detail dripsHubAdmin address)",
+    "Caller":                       "$(deployment_detail caller address)",
+    "AddressDriver":                "$(deployment_detail addressDriver address)",
+    "AddressDriver logic":          "$(deployment_detail addressDriverLogic address)",
+    "AddressDriver admin":          "$(deployment_detail addressDriverAdmin address)",
+    "AddressDriver ID":             "$(deployment_detail addressDriverId uint32)",
+    "NFTDriver":                    "$(deployment_detail nftDriver address)",
+    "NFTDriver logic":              "$(deployment_detail nftDriverLogic address)",
+    "NFTDriver admin":              "$(deployment_detail nftDriverAdmin address)",
+    "NFTDriver ID":                 "$(deployment_detail nftDriverId uint32)",
+    "ImmutableSplitsDriver":        "$(deployment_detail immutableSplitsDriver address)",
+    "ImmutableSplitsDriver logic":  "$(deployment_detail immutableSplitsDriverLogic address)",
+    "ImmutableSplitsDriver admin":  "$(deployment_detail immutableSplitsDriverAdmin address)",
+    "ImmutableSplitsDriver ID":     "$(deployment_detail immutableSplitsDriverId uint32)",
     "Commit hash":                  "$(git rev-parse HEAD)"
```

Comment:
Since we trying to make this perfect here. We could also add a deployment timestamp. :)

Reply:
It's doable, but do we need it? When the JSON is read, the contracts are already deployed. If you're just curious, all the data is on-chain. It's not critical information from the trust and security perspective either.

Reply:
Yes, of course, you can fetch it from the transactions. If we end up multiple of this json files (different deployment times) it might be nice to have it.

Reply:
That's a good point. It doesn't require blockchain-perfect transaction timestamp, so it can be much simpler. I've added it using a regular time from the local machine, it's very easy and for comparing different JSONs it should be enough.